scripts/deploy/update_version_numbers.rb

```diff
@@ -27,3 +28,23 @@ def update_gradle_properties()
       "VERSION_NAME=#{@version}",
   )
 end
+
+def update_changelog()
+  done = false
+  final_lines = []
+
+  File.foreach('CHANGELOG.md') do |line|
+    is_line = line.include? "XX.XX.XX"
+    final_lines.append(line)
+
+    date = Time.now.strftime("%Y-%m-%d")
+
+    if is_line
```

Comment:
Should we break once we hit this?

Reply:
I guess not since we rewrite the whole file.

Reply:
I just updated the implementation of this to re-use the `replace_in_file` function so that we no longer do this. I should have just done that from the start!